Entities/Natural/Seed/Seed.as

```diff
@@ -136,4 +136,25 @@ void onTick(CBlob@ this)
 			}*/
 		}
 	}
+
+	if (isServer())
```

Comment:
checkout FallOnNoSupport.as. You might want to look into using that script. Or using onBlobCollapse hook.